src/zope/app/debug/debug.py

```diff
@@ -191,12 +198,11 @@ def fbreak(db, meth):
     code = meth.__code__
     lineno = getlineno(code)
     filename = code.co_filename
-    db.set_break(filename,lineno)
-
+    db.set_break(filename, lineno)
 
 
 try:
     from codehack import getlineno
-except:
+except ImportError:
```

Comment:
We do not have `codehack` declared as dependency anywhere and the last release is from 2007 so we should default to `code.co_firstlineno` in all cases. It would be nice to have it defined before `fbreak()` or included directly.

From https://pypi.org/project/codehack/:
```
This module include 2 features,
    one is extension (written in C) to manipulate code objects, and the other is utility modules work around code objects. Both are in alpha phase. Don’t use to important project.
```

Reply:
Nice idea, implemented in https://github.com/zopefoundation/zope.app.debug/pull/2/commits/46b163277d309e0ce910228473b696427b8ca726